card/lib/card/log.rb

```diff
@@ -194,7 +196,7 @@ def print_log
 
       def text_log
         @@log.each do |entry|
-          Rails.logger.wagn entry.to_s! if entry.valid
+          Rails.logger.send @log_level, entry.to_s! if entry.valid
```

Comment:
`Logger` has changed and I couldn't find a easy way to keep the custom wagn log level. Instead of that there is now a log_level option for the performance logger.